--- gdm-password.debiantrixie.orig	2025-01-18 16:21:35.803518905 -0300
+++ gdm-password	2025-01-18 17:56:59.527824585 -0300
@@ -1,6 +1,8 @@
 #%PAM-1.0
 auth    requisite       pam_nologin.so
 auth	required	pam_succeed_if.so user != root quiet_success
+auth    [success=ignore default=1]  pam_script.so dir=/etc/pam.d/scripts/is-session-locked
+@include common-pinpwd
 @include common-auth
 auth    optional        pam_gnome_keyring.so
 @include common-account
